generators/server/files.js

```diff
@@ -940,14 +940,16 @@ const serverFiles = {
     ],
     serverJavaUserManagement: [
         {
-            condition: generator => (generator.skipUserManagement && generator.authenticationType === 'oauth2') || (!generator.skipUserManagement && generator.databaseType === 'sql'),
```

Comment:
for microservices ? is this not required ?

Reply:
I updated it since your comment to load into the gateway, but no it's not required.  Currently, the OAuth2 gateway doesn't load the users.csv and instead syncs the users from Keycloak/Okta on that account's first login.  This PR updates the OAuth2 gateways to import the `anonymoususer` and `system` accounts.  

Same thing with Authorities, except the Java tests check for them so ROLE_USER and ROLE_ADMIN are loaded.